aunt-leahs-app/src/components/customButton.jsx

```diff
@@ -38,23 +38,22 @@ const useStyles = makeStyles({
 
 // Two different sizes: small and large 
 // Two different colors: primary (teal) and secondary (red)
-const CustomButton = ({size, color, onClick, children}) => {  
+const CustomButton = ({ size, color, onClick, children }) => {
     const classes = useStyles();
-    
+
     return (
-    <Button size={size} color={color} variant='contained' onClick={onClick} 
-    classes={{
-        root: classes.root,
-        label: classes.label,
-        containedPrimary: classes.containedPrimary,
-        containedSecondary: classes.containedSecondary,
-        containedSizeSmall: classes.containedSizeSmall,
-        containedSizeLarge: classes.containedSizeLarge
-        }}>
-        {children}
-    </Button>
+        <Button size={size} color={color} variant='contained' onClick={onClick}
+            classes={{
+                root: classes.root,
+                label: classes.label,
+                containedPrimary: classes.containedPrimary,
+                containedSecondary: classes.containedSecondary,
+                containedSizeSmall: classes.containedSizeSmall,
+                containedSizeLarge: classes.containedSizeLarge
+            }}>
+            {children}
+        </Button>
     );
 };
 
 export default CustomButton;
-
```

Comment:
I think Cody mentioned something about having a blank line at the end of every file?

Reply:
There is a blank line there, it just doesn't show on github. As long as there is a not a warning, that is good.